scirpy/io/_io.py

```diff
@@ -443,3 +527,13 @@ def read_bracer(path: Union[str, Path]) -> AnnData:
         tmp_ir_cell.add_chain(tmp_chain)
 
     return from_ir_objs(bcr_cells.values())
+
+def read_dandelion(Dandelion, import_all = False):
```

Comment:
```suggestion
def read_dandelion(dandelion, import_all = False):
```

only class names should start with upper case, not instances thereof